README.md

```diff
@@ -2,11 +2,36 @@
 
 NOTE: The project is under development.
 
+## PREREQUISITES
 
-## Deploy Onload Operator
+1) Go version 1.21+
```

Comment:
Is Go actually a requirement?
I think it is only necessary when running tests or build outside of containers.